zerver/tests/test_outgoing_webhook_interfaces.py

```diff
@@ -8,21 +8,24 @@
 from zerver.lib.outgoing_webhook import GenericOutgoingWebhookService, \
     SlackOutgoingWebhookService
 from zerver.lib.test_classes import ZulipTestCase
-from zerver.models import Service
+from zerver.models import Service, get_realm, get_user
 
 class TestGenericOutgoingWebhookService(ZulipTestCase):
 
     def setUp(self) -> None:
         self.event = {
             u'command': '@**test**',
             u'message': {
-                'content': 'test_content',
-            }
+                'content': '@**test**',
+            },
+            u'trigger': 'mention',
         }
         self.handler = GenericOutgoingWebhookService(service_name='test-service',
                                                      base_url='http://example.domain.com',
                                                      token='abcdef',
-                                                     user_profile=None)
+                                                     user_profile=get_user(
+                                                         "outgoing-webhook@zulip.com",
+                                                         get_realm("zulip")))
```

Comment:
Let's move this to a `self.bot_user = ` line earlier in `setUp`; then other test code can access it.

Reply:
(Obviously, it doesn't need to right now, but structurally, it's cleaner)